snabble/Android-SDK
# Add new dependency management and release on Tag

### What's new?

All gradle files have been converted to gradle.kts files. The dependencies are now referenced from the version catalog and the plugin to maintain these has been added. Also a release on Tag is now available.

### How to Test?

* Run each sample and try out different things to find any update related bugs.
* Enter 'export VERSION_CODE=9999' into the terminal and deploy the project locally. The new deployed package should have version 9999 if env var is exported otherwise it should be def.
